internal/ethapi/api.coreth.go

```diff
@@ -67,15 +64,23 @@ func (s *EthereumAPI) SuggestPriceOptions(ctx context.Context) (*PriceOptions, e
 		return nil, nil
 	}
 
+	cfg := s.b.PriceOptionsConfig()
+	bigSlowFeePerc := new(big.Int).SetUint64(cfg.SlowFeePercentage)
```

Comment:
nit: seems we can rename to `bigSlowFeePercent` and `SlowFeePercent` to strike a balance between brevity and avoiding abbreviations.
perhaps even `SlowFeeMultiplierPercent`?